module/metrics/gossipsub_score.go

```diff
@@ -72,17 +66,17 @@ func NewGossipSubScoreMetrics(prefix string) *GossipSubScoreMetrics {
 			Subsystem: subsystemGossip,
 			Name:      prefix + "gossipsub_ip_collocation_factor_score",
 			Help:      "ip collocation factor from gossipsub peer scoring",
-			Buckets:   gossipSubScoreBuckets,
+			Buckets:   []float64{10, 100, 1000},
 		},
 	)
 
 	gs.timeInMesh = *promauto.NewHistogramVec(
 		prometheus.HistogramOpts{
 			Namespace: namespaceNetwork,
 			Subsystem: subsystemGossip,
-			Name:      prefix + "gossipsub_time_in_mesh_score",
-			Help:      "time in mesh from gossipsub scoring",
-			Buckets:   gossipSubScoreBuckets,
+			Name:      prefix + "gossipsub_time_in_mesh_quantum_count",
+			Help:      "time in mesh from gossipsub scoring; number of the time quantum a peer has been in the mesh",
+			Buckets:   []float64{1, 24, 168}, // 1h, 1d, 1w with quantum of 1h
```

Comment:
What does "quantum" mean in this context?

Reply:
The atomic unit of time to gauge the availability of based on. For example, when setting the quantum to 1 hour, a node's availability can be an integer multiple of hour; but not a fraction.